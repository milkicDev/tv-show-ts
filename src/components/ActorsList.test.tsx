import React from 'react';
import renderer from 'react-test-renderer'
import Actor from '../models/Actor';
import ActorLists from './ActorsList';
import ShowsApi from '../api/Shows';

it('render app', async () => {
  let render, actors: Actor[];

  await ShowsApi.getCast(1)
    .then((res) => {
      actors = res;
      render = renderer.create(<ActorLists actor={actors[0]} />).toJSON();
    })
    .catch((error) => console.error(error));

  expect(render).toMatchSnapshot();
})
