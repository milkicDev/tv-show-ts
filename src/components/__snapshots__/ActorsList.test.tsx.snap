// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<div
  className="actor"
>
  <span
    className="img"
  >
    <img
      alt="Mike Vogel"
      src="https://static.tvmaze.com/uploads/images/medium_portrait/0/1815.jpg"
    />
  </span>
  <span>
    Mike Vogel
  </span>
  <span>
    Dale "Barbie" Barbara
  </span>
</div>
`;
